### DISCUSSION TITLE
Quotes around the empty string and  adding specified number of spaces if value is empty

### DISCUSSION THREAD

**Discussion by lmoparthi** [JoshClose/CsvHelper#1772] [Q&A]
Hi 
I could not figure it out 
1. how to remove the quotes around the string fields in writing to csv file 
2. I have to add specified number of zeros/spaces if value is empty or null.
when i am writing into csv file with delimiter semi colon ;, . 
presently my files looks like below 

DE;0;;"          ";;;;;;;"000757000000        ";"KERKER                             ";

I want to achieve something like below with no quotes around the strings and 

DE;0;00000;            ;000000;00000;00000;00000;00000;00000;000757000000        ;KERKER                             ;

Any help would be greatly appreciated.
Thank you so much in advance.

Thank you

**Comment by JoshClose** [comment]
There is a config option `ShouldQuote` that you can use to say when a field should be quoted. If you never want them quoted, you can use `Mode = CsvMode.NoEscape`. Not writing quotes might make it impossible to read back in though, so I'd recommend another approach.

You could write a customer converter to write spaces or zeros. This converter would also detect this when reading back in too.
